### PR TITLE
test: make Test_mergeEnvSlice test case happy

### PR DESCRIPTION
**Pull request by HusterWan**
Signed-off-by: Michael Wan <zirenwan@gmail.com>

### Ⅰ. Describe what this PR did

Now we try TOP(Test-oriented programming), so in order making my test case happy

Just a joke 🤣

### CODE REVIEW

**Comment by chuanchang** [comment]
LGTM

File: daemon/mgr/container_utils_test.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func Test_mergeEnvSlice(t *testing.T) {
 				newEnv: []string{"a=b"},
 				oldEnv: []string{"c=d"},
 			},
-			want:    []string{"c=d", "a=b"},
```
Comment (Ace-Tang) [inline daemon/mgr/container_utils_test.go]: I do not advise this way to fix, try to match every value in slice. But consider you have sort before, it also work.

Comment (fuweid) [reply daemon/mgr/container_utils_test.go]: Agree. we should sort the slice before we check.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**